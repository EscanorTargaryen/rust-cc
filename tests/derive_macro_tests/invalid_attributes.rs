use rust_cc::*;

#[derive(Trace, Finalize)]
#[rust_cc]
#[rust_cc = ""]
struct MyStruct1 {
}

#[derive(Trace, Finalize)]
struct MyStruct2 {
    #[rust_cc]
    #[rust_cc = ""]
    a: (),
}

#[derive(Trace, Finalize)]
#[rust_cc]
#[rust_cc = ""]
enum MyEnum1 {
}

#[derive(Trace, Finalize)]
enum MyEnum3 {
    #[rust_cc]
    #[rust_cc = ""]
    A(#[rust_cc] #[rust_cc = ""] i32),
    #[rust_cc]
    #[rust_cc = ""]
    B {
        #[rust_cc]
        #[rust_cc = ""]
        b: i32,
    }
}

fn main() {
}
