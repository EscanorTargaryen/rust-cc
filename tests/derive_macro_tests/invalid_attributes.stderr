error: Invalid attribute
 --> tests/derive_macro_tests/invalid_attributes.rs:4:3
  |
4 | #[rust_cc]
  |   ^^^^^^^

error: Invalid attribute
 --> tests/derive_macro_tests/invalid_attributes.rs:5:3
  |
5 | #[rust_cc = ""]
  |   ^^^^^^^^^^^^

error: Invalid attribute
  --> tests/derive_macro_tests/invalid_attributes.rs:11:7
   |
11 |     #[rust_cc]
   |       ^^^^^^^

error: Invalid attribute
  --> tests/derive_macro_tests/invalid_attributes.rs:12:7
   |
12 |     #[rust_cc = ""]
   |       ^^^^^^^^^^^^

error: Invalid attribute
  --> tests/derive_macro_tests/invalid_attributes.rs:17:3
   |
17 | #[rust_cc]
   |   ^^^^^^^

error: Invalid attribute
  --> tests/derive_macro_tests/invalid_attributes.rs:18:3
   |
18 | #[rust_cc = ""]
   |   ^^^^^^^^^^^^

error: Invalid attribute
  --> tests/derive_macro_tests/invalid_attributes.rs:26:9
   |
26 |     A(#[rust_cc] #[rust_cc = ""] i32),
   |         ^^^^^^^

error: Invalid attribute
  --> tests/derive_macro_tests/invalid_attributes.rs:26:20
   |
26 |     A(#[rust_cc] #[rust_cc = ""] i32),
   |                    ^^^^^^^^^^^^

error: Invalid attribute
  --> tests/derive_macro_tests/invalid_attributes.rs:30:11
   |
30 |         #[rust_cc]
   |           ^^^^^^^

error: Invalid attribute
  --> tests/derive_macro_tests/invalid_attributes.rs:31:11
   |
31 |         #[rust_cc = ""]
   |           ^^^^^^^^^^^^

error: Invalid attribute
  --> tests/derive_macro_tests/invalid_attributes.rs:24:7
   |
24 |     #[rust_cc]
   |       ^^^^^^^

error: Invalid attribute
  --> tests/derive_macro_tests/invalid_attributes.rs:25:7
   |
25 |     #[rust_cc = ""]
   |       ^^^^^^^^^^^^

error: Invalid attribute
  --> tests/derive_macro_tests/invalid_attributes.rs:27:7
   |
27 |     #[rust_cc]
   |       ^^^^^^^

error: Invalid attribute
  --> tests/derive_macro_tests/invalid_attributes.rs:28:7
   |
28 |     #[rust_cc = ""]
   |       ^^^^^^^^^^^^
